File: C&C++/Ansh Gulati/Day-4_06-07-23/Ques-1.c
/* Ques- Write a program to determine the type of a triangle based on the 
lengths of its sides and print the result (e.g., equilateral, isosceles, 
scalene).Take the input of the length of sides from the User. */

#include<stdio.h>
int main()
{
    int side1;
    int side2;
    int side3;
    printf("Enter the length of first side of the triangle: ");
    scanf("%d",&side1);
    printf("Enter the length of second side of the triangle: ");
    scanf("%d",&side2);
    printf("Enter the length of third side of the triangle: ");
    scanf("%d",&side3);
    if(side1==side2 && side2 == side3)
    {
        printf("\nThe given Triangle is Equilateral");
    }
    else if(side1==side2 || side2==side3 || side1==side3)
    {
        printf("\nThe given Triangle is Isosceles");
    }
    else
    {
        printf("\nThe given Triangle is Scalene");  
    }
}

File: Python/Ansh Gulati/Day-3_05-07-23/Ques-1.py
# Ques- Write a Python program that takes three numbers as input and prints the maximum of the three.

num1=int(input("Enter the value of first number: "))
num2=int(input("Enter the value of second number: "))
num3=int(input("Enter the value of third number: "))

if(num1>num2):
    if (num1 > num3) :
        print("The Maximum of three numbers is: ",num1)
    else:
        print("The Maximum of three numbers is: ",num3)
else:
    if (num2 > num3) :
        print("The Maximum of three numbers is: ",num2)
    else:
        print("The Maximum of three numbers is: ",num3)
